src/index.js

```diff
@@ -34,7 +33,6 @@ export const HANDLERS = {
   'broken-backlinks': noopHandler,
   'broken-backlinks-external': brokenBacklinksExternal,
   'broken-backlinks-internal': brokenBacklinksInternal,
-  experimentation,
```

Comment:
Here you have to keep it, but make it call (noopHandler)
`experimentation: noopHandler`

Reply:
done.